src/compiler/crystal/macros/methods.cr

```diff
@@ -1654,6 +1656,18 @@ module Crystal
       end
     end
 
+    def self.all_includers(type)
+      case type
+      when NonGenericModuleType, GenericModuleType
+        return empty_no_return_array unless types = type.raw_including_types
```

Comment:
The main logic should not be hidden in a suffix condition. Please change this to either two separate expressions:
```suggestion
        types = type.raw_including_types
        return empty_no_return_array unless types
```
Alternatively, change the order of condition and effect:
```suggestion
        types = type.raw_including_types || return empty_no_return_array
```